usr/lib/common/loadsave.c

```diff
@@ -1604,20 +1675,14 @@ CK_RV load_public_token_objects_old(STDLL_TokData_t *tokdata)
     CK_ULONG_32 size;
     size_t read_size;
 
-    sprintf(iname, "%s/%s/%s", tokdata->data_store, PK_LITE_OBJ_DIR,
-            PK_LITE_OBJ_IDX);
-
-    fp1 = fopen(iname, "r");
+    fp1 = open_token_object_index(iname, sizeof(iname), tokdata, "r");
     if (!fp1)
         return CKR_OK;          // no token objects
 
     while (fgets(tmp, 50, fp1)) {
```

Comment:
again this 50 value out of the blue - check for replace with some sizeof(tmp)

Reply:
Done.  Actually, checks for fgets were not done in these changes.  They are still pending.  But due to the size of the changes I wanted to upload that amount already in advance.